src/sentry/auth/idpmigration.py

```diff
@@ -85,11 +104,8 @@ def verify_account(key: str) -> bool:
     :param key: the one-time verification key
     :return: whether the key is valid
     """
-    cluster = redis.clusters.get("default").get_local_client_for_key(_REDIS_KEY)
-
-    verification_key = get_redis_key(key)
-    verification_value_byte = cluster.hgetall(verification_key)
-    if not verification_value_byte:
+    verification_value = get_redis_key(key)
+    if not verification_value:
```

Comment:
should we also clear out the redis value once we've verified it?

Reply:
do we need to? it will clear within 10 mins, but is there harm leaving it until the 10 mins is up?

Reply:
not sure if related, what happens if they click on the link again after they've already verified, logged in, and created a new identity?

Reply:
It seems like nothing unexpected happens. it wouldn't go thru the verification flow since it no longer enters handle_unknown_identity

Reply:
alright, sounds good.